integration-tests/pom.xml

```diff
@@ -90,6 +90,12 @@
             <artifactId>configuration-loader-valve</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.wso2.appserver</groupId>
+            <artifactId>jpa-contacts-database</artifactId>
+            <classifier>classes</classifier>
```

Comment:
because we need to have classes of this sample in order to run the test, to get the classes we need to have a classifier since this is a war dependency.